recipe-server/client/control/components/ExtensionForm.js

```diff
@@ -0,0 +1,137 @@
+import { Map } from 'immutable';
+import React, { PropTypes as pt } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { initialize, reduxForm } from 'redux-form';
+
+import QueryExtension from './data/QueryExtension';
```

Comment:
i didn't think we'd settled on a standard, but I'm happy to switch. I think we should probably make a decision one way or the other and stick with it.